src/components/products/list/ProductsList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import testProducts from '../ProductsData'
+import SimpleList from '../../common/simpleList/SimpleList'
+import { withRouter } from 'react-router-dom'
+
+class ProductsList extends Component {
+    productsListObjects = testProducts.map(product => {
```

Comment:
pull develop into your branch and use apiService to load data already. You can look at CustomersList component to see the example